safe_control_gym/envs/benchmark_env.py

```diff
@@ -298,7 +298,7 @@ def _randomize_values_by_info(self,
         return randomized_values
 
     @abstractmethod
-    def _setup_symbolic(self):
+    def _setup_symbolic(self, prior_prop={}, **kwargs):
```

Comment:
does this return the mode? and the docstring should include args and returns

Reply:
it doesn't return any, but assign/re-assign the prior model to env.symbolic (so anytime env and ctrl will still have the same prior and env will get the correct quadratic cost from _get_reward()). yse i will add the docstring